MathPortal/MathPortal/Source/Data/User.swift

```diff
@@ -93,7 +105,10 @@ class User: ParseObject {
     }
     
     static func emailIsAlreadyTaken(email: String?, completion: @escaping ((_ state: Bool?, _ error: Error? ) -> Void)) {
-        guard let email = email else { completion(nil,NSError()); return }
+        guard let email = email else {
+            completion(nil,NSError())
+            return
```

Comment:
`generateQueryWithEmail(email)?` is still optional. It also needs to be unwrapped or completion will not be called.